Storage/Storage/Tools/StorageType+Extensions.swift

```diff
@@ -82,6 +82,21 @@ public extension StorageType {
         return firstObject(ofType: OrderStatsItem.self, matching: predicate)
     }
 
+    /// Retrieves all of the Stores OrderStatuses for the provided siteID.
+    ///
+    public func loadOrderStatuses(siteID: Int) -> [OrderStatus]? {
```

Comment:
@mindgraffiti I think this should be `loadOrderStatii()` 🤣🤣🤣

JK — I never get to correct your grammar/typos, so -

Reply:
🤔 I think you have me there!